sdk/storage_datalake/src/clients/data_lake_client.rs

```diff
@@ -1,8 +1,10 @@
-use crate::authorization_policy::AuthorizationPolicy;
+// use crate::bearer_token_authorization_policy::BearerTokenAuthorizationPolicy;
```

Comment:
Can we remove this? I think the TODO below is sufficient, and we don't need to keep this.

Reply:
Done